readme.md

```diff
@@ -1,23 +1,29 @@
-# ![](https://ga-dash.s3.amazonaws.com/production/assets/logo-9f88ae6c9c3871690e33280fcf557f33.png) Project #3: Building a SPA with an Angular Front-end and express/Rails backend
+# ![](https://ga-dash.s3.amazonaws.com/production/assets/logo-9f88ae6c9c3871690e33280fcf557f33.png) Project #3: Building a SPA with an Angular Front-end and Rails Back-end
 
 ### Overview
 
-While your last project taught you how to get started with Ruby, SQL, & Ruby on Rails, this project you'll be building something exciting combining **Angular front-end with a backend of your choice**
+While your last project taught you how to get started with Ruby and Rails, this project will have you building something exciting by combining an **Angular front-end with a Rails back-end.**
 
-Your instructors will partner you with other classmates to architect, design, and collaboratively build an app with two major components:
+Your instructors will partner you with other classmates to architect, design and collaboratively build an app with two major components:
 
 1. An API of your own design, built using Rails, which serves JSON.
-2. Front-end Angular code that updates the UI, and makes requests to the API using AJAX.
+2. Front-end Angular code that updates the UI and makes requests to the API using AJAX.
 
 **This is meant to push you both technically and collaboratively.**  You will, almost certainly, be joining a development team during your career.  Working collaboratively is a learned skill, just like programming. **It's important to learn how to work together.**
 
 ---
 
 ### Requirements
-- This app must contain a backend API with at least 2 models. No associations are required.
-- This app must use an angular front end that leverages the backend API in the above requirement.
+- This app must contain a back-end API with at least 2 models. No associations are required.
+- This app must use an Angular front-end that leverages the backend API in the above requirement.
 - Work hard and have fun!
 
```

Comment:
Would recommend adding a reference to the project 3 evaluation form:  "Stuck? Reference the Project 3 evaluation form to see how your projects will be graded." link: https://github.com/ga-wdi-exercises/project3/blob/wdi14/evaluation.md